src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -0,0 +1,852 @@
+     */
+    @Argument(fullName="aggregate",
+            shortName = "aggregate", doc="Additional raw input variants to be used in building the model",
+            optional=true)
+    private List<FeatureInput<VariantContext>> aggregate = new ArrayList<>();
+
+    /**
+     * Any set of VCF files to use as lists of training, truth, or known sites.
+     * Training - The program builds the Gaussian mixture model using input variants that overlap with these training sites.
+     * Truth - The program uses these truth sites to determine where to set the cutoff in VQSLOD sensitivity.
+     * Known - The program only uses known sites for reporting purposes (to indicate whether variants are already known or
+     * novel). They are not used in any calculations by the algorithm itself.
+     * Bad - A database of known bad variants can be used to supplement the set of worst ranked variants (compared to the
+     * Gaussian mixture model) that the program selects from the data to model "bad" variants.
+     */
+    @Argument(fullName="resource",
+            shortName = "resource",
+            doc="A list of sites for which to apply a prior probability of being correct but which aren't used by the algorithm (training and truth sets are required to run)",
+            optional=false)
+    private List<FeatureInput<VariantContext>> resource = new ArrayList<>();
+
+    /////////////////////////////
+    // Outputs
+    /////////////////////////////
+    @Argument(fullName= StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName=StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="The output recal file used by ApplyRecalibration", optional=false)
+    private String output;
+
+    @Argument(fullName="tranches_file", shortName="tranchesFile", doc="The output tranches file used by ApplyRecalibration", optional=false)
+    private String TRANCHES_FILE;
+
+    /////////////////////////////
+    // Additional Command Line Arguments
+    /////////////////////////////
+    /**
+     * The expected transition / transversion ratio of true novel variants in your targeted region (whole genome, exome, specific
+     * genes), which varies greatly by the CpG and GC content of the region. See expected Ti/Tv ratios section of the GATK best
+     * practices documentation (http://www.broadinstitute.org/gatk/guide/best-practices) for more information.
+     * Normal values are 2.15 for human whole genome values and 3.2 for human whole exomes. Note
+     * that this parameter is used for display purposes only and isn't used anywhere in the algorithm!
+     */
+    @Argument(fullName="target_titv",
+            shortName="titv",
+            doc="The expected novel Ti/Tv ratio to use when calculating FDR tranches and for display on the optimization curve output figures. (approx 2.15 for whole genome experiments). ONLY USED FOR PLOTTING PURPOSES!",
+            optional=false)
+    private double TARGET_TITV = 2.15;
```

Comment:
Whoops, this arg was `required=false` in GATK3, so it should be `optional=true` here.